app.py

```diff
@@ -137,8 +137,10 @@ def mount_sidebar(self, executor):
             for channel in self.store.state.channels
         ]
         dms = []
-        dm_users = self.store.state.dms[:15]
-        for dm in dm_users:
+        max_users_sidebar = self.store.config['sidebar']['max_users']
+        listed_user = 0
+
+        for dm in self.store.state.dms:
```

Comment:
I think `self.store.state.dms[:max_users_sidebar]` can do the same thing as taking the `n` first users.

Reply:
I can see it, but i prefer to use `break` to limit the items. This solution could work when we implement solution to support https://github.com/haskellcamargo/sclack/issues/36.
How do you feel about that?

Reply:
Mount sidebar is called once when the application start or when you switch workspace. The sidebar is rendered as soon as we have the plain list of channels (with not all necessary data). To support #36, we need to dispatch a serie of asynchronous tasks to get the information about a channel (if it is starred and if there are unread messages), so we can hide the read ones. Both he Slack web client and Sclack set the "white/bold" highlights to these channels only some time after rendering the list itself, so we couldn't implement it there because the sidebar is a pre-render. Slack web, however, uses local storage to cache the already read channels and give a good visual feedback, so, in this situation, I vote for less complexity in this method (using `:n`) and implementing like Slack web when on #36.

Reply:
Okie, understood your idea. Let me update that logic